File: Miyazawa/module_oriented/service.rb
module Service
    def sign_up
    a={}
    puts "メールアドレスを入力してください"
    mail_adress = gets.chomp
    a['mail_address'] = mail_adress
    puts "電話番号を入力してください"
    phone_number = gets.chomp
    a['phone_number'] = mail_adress

    puts "名字を入力してください"
    name_kanji = gets.chomp
    a['name_kanji'] = mail_adress

    puts "名前を入力してください"
    name_kana = gets.chomp
    a['name_kana'] = mail_adress

    puts "性別を選択してください 男性or女性"
    sex_division = gets.chomp
    a['sex_division'] = mail_adress

    puts "生年月日を入力してください"
    birth_day = gets.chomp
    a['birth_day'] = mail_adress
    @members[@members.length] = a
    puts "会員登録が完了しました"
  end

  def exit
    puts "サービスを終了します"
  end
end

File: Yamaura/ObjectOrientedArchitecture/ObjectOrientedArchitecture/src/jp/co/netprotections/Service.java
package jp.co.netprotections;

import java.util.Scanner;



public class Service {


	public static void serviceRegister() {
	    System.out.println("２．サービス利用登録/停止");
	    System.out.print("情報を変更したい会員のidを入力ください");
	    Scanner cr10 = new Scanner(System.in);
	    int selectedId = cr10.nextInt();
	    System.out.println(Member.memberList.get(selectedId).id);
	    System.out.println(Member.memberList.get(selectedId).mail_address);
	    System.out.println(Member.memberList.get(selectedId).phone_number);
	    System.out.println(Member.memberList.get(selectedId).name_kanji);
	    System.out.println(Member.memberList.get(selectedId).name_kana);
	    System.out.println(Member.memberList.get(selectedId).sex_division);
	    System.out.println(Member.memberList.get(selectedId).birth_day);
	    System.out.println(Member.memberList.get(selectedId).address);
	    System.out.println(Member.memberList.get(selectedId).fufululuStatus);
	    System.out.println("１．フフルル：");
	    System.out.println("２．NP後払い：");
	    System.out.println("３．atone：");
	    System.out.print("サービス登録/停止したい番号を入力ください：");
	    Scanner cr11 = new Scanner(System.in);
	    int reqNumber = cr11.nextInt();
	    if (reqNumber == 1) {
	      if (Member.memberList.get(selectedId).fufululuStatus == false) {
	        Member.memberList.get(selectedId).fufululuStatus = true;
	        System.out.println("フフルルサービスを登録しました");
	      }
	      else {
	        Member.memberList.get(selectedId).fufululuStatus = false;
	        System.out.println("フフルルサービスを停止しました");
	      }
	    } else if (reqNumber == 2) {
	      if (Member.memberList.get(selectedId).atobaraiStatus == false) {
	        Member.memberList.get(selectedId).atobaraiStatus = true;
	        System.out.println("NP後払いサービスを登録しました");
	      }
	      else {
	        Member.memberList.get(selectedId).atobaraiStatus = false;
	        System.out.println("NP後払いサービスを停止しました");
	      }
	    } else if (reqNumber == 3) {
	      if ( Member.memberList.get(selectedId).atoneStatus == false) {
	    	 Member.memberList.get(selectedId).atoneStatus = true;
	    	 System.out.println("atoneサービスを登録しました");
	      }
	      else {
	        Member.memberList.get(selectedId).atoneStatus = false;
	        System.out.println("atoneサービスを停止しました");
	      }
	    } else {
	      System.out.println("1~3の番号を入力ください");

	    }
	}



}
